docs/reference/glossary.md

```diff
@@ -0,0 +1,43 @@
+# Glossary
+
```

Comment:
content delivery network (CDN) ?

Reply:
I only imported the ones already existing in conda-forge.org for completeness, but we are not focusing on this right now. Issue welcome :D